src/actinia_core/README.md

```diff
@@ -7,7 +7,7 @@ This mainly concerns the plugins:
 * [actinia_satallite_plugin](https://github.com/mundialis/actinia_satellite_plugin) (satellite)
 * [actinia-metadata-plugin](https://github.com/mundialis/actinia-metadata-plugin) uses no sources of actinia_core
 * [actinia-stac-plugin](https://github.com/mundialis/actinia-stac-plugin) (stac)
-* [actinia-tiling-plugin](https://github.com/mundialis/actinia-tiling-plugin) (tiling TODO: add imports)
+* [actinia-tiling-plugin](https://github.com/mundialis/actinia-tiling-plugin) (tiling)
 * [actinia-api](https://github.com/mundialis/actinia-api) (api)
 
 The modules inside common folder are not only commonly used by actinia but also by actinia plugins.
```

Comment:
Please also add Modules to above table (I am not allowed to comment directly there as you didn't change anything there. I didn't check much, only saw that eg `from actinia_core.core.common.process_chain import ProcessChainConverter` is missing.

Reply:
They are added to the table